src/main/webapp/map.js

```diff
@@ -39,8 +40,10 @@ function createMap() {
   }
 }
 
-// Request nearby coffee shop info
-// Unit of radius: metres. Maximum allowed is 50000 metres
+/**
+/* Request nearby coffee shop info
+/* Unit of radius: Metres. Maximum allowed is 50000 metres
```

Comment:
Add a comment as to whether the 500m max is due to an API restriction or a TYD Coffee restriction.